docs/_frontend/src/models/Sidebar/Sidebar.js

```diff
@@ -1,18 +1,34 @@
-import Combobox from '../Combobox';
 import Radio from '../Radio/Radio';
 import RadioGroup from '../RadioGroup';
+import SearchCombobox from '../SearchCombobox/SearchCombobox';
 import styles from './Sidebar.module.css';
+import { SEARCH_ORIGIN, getSearchString } from '../common';
+
+const toAutoCompleteSearchString = (input, field, params) => {
```

Comment:
We could use here something like:
```js
const toAutoCompleteSearchString = (input, field, params) => getSearchString({search: input, field, ...params});
```
It allows not to concatenate strings manually.

Reply:
Thanks .. Implemented